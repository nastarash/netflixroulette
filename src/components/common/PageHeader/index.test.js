import React from 'react';
import { PageHeader } from './index';

describe('<PageHeader />', () => {
  test('should be rendered correctly', () => {
    const pageHeader = shallow(
      <PageHeader />
    );
    expect(pageHeader).toMatchSnapshot();
  });
});