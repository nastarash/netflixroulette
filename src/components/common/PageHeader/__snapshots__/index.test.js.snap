// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageHeader /> should be rendered correctly 1`] = `
<header>
  <div
    className="page-header"
  >
    <div
      className="page-header__background"
    />
    <div
      className="page-header__container"
    >
      <Logo />
      <SearchPanel />
    </div>
  </div>
  <Connect(SearchResults) />
</header>
`;
